linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/bash/shared.sh

```diff
@@ -1,7 +1,8 @@
 # platform = multi_platform_wrlinux,multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
 
-# Find which files have incorrect hash (not in /etc, because there are all system related config. files) and then get files names
-files_with_incorrect_hash="$(rpm -Va | grep -E '^..5.* /(bin|sbin|lib|lib64|usr)/' | awk '{print $NF}' )"
```

Comment:
The `--noconfig` option is not available in RHEL-6, the grep approach needs to be kept for this product.

Reply:
Then we have a problem in the ansible remediation as well: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/ansible/shared.yml#L17

Reply:
Well noted.
As it is now it doesn't work for `rhel6`, I would be fine just conditionally adding`--config` if not building for `rhel6`.

At this moment we don't need to be as elaborate as the Bash remediation.

Reply:
Ok, I will get back to this soon when I have time. Thanks.

Reply:
I have updated the branch but it is not syncing with the PR, I've contacted github support team. Meanwhile you can take a look at the changes here: https://github.com/ggbecker/content/commit/0458852e38a0817bb6d06fcad8ddd3673ec37d23

Reply:
Now the PR is updated

Reply:
> At this moment we don't need to be as elaborate as the Bash remediation.

Sorry I didn't catch this earlier, I meant to keep the Bash remediation grepping the paths in `rhel6`.
My concern is that removing the paths in the grep would cause too much false positives.

Reply:
I see. It makes sense. I'll update accordingly.

Reply:
I have updated the bash remediation with a slightly different approach so it's more readable from the developer's perspective.